synapse/models/orgs.py

```diff
@@ -734,6 +748,44 @@ def getModelDefs(self):
                     ('org', ('ou:org', {}), {}),
                     ('name', ('ou:name', {}), {}),
                 )),
+
+                ('ou:asset', {}, (
+                    ('org', ('ou:org', {}), {
```

Comment:
You have :org / ou:org as "The organization that owns the asset" and also :owner / ps:contact as ''contact information of the asset owner". These seem either duplicative or else I'm not sure of the distinction / we need better docstrings?

Reply:
The difference is identifying a contact for a person "responsible" for the asset vs the org which is tracking the asset. One could assume that :owner::org should link up, but i think the direct property is worth it for clarity. Thoughts?

Reply:
try to clarify the doc string :+1: